forc/src/ops/forc_build.rs

```diff
@@ -57,16 +57,24 @@ pub fn build(command: BuildCommand) -> Result<pkg::Compiled> {
 
     // If necessary, construct a new build plan.
     let plan: pkg::BuildPlan = plan_result.or_else(|e| -> Result<pkg::BuildPlan> {
-        println!("  Creating a new `Forc.lock` file");
```

Comment:
I believe it's only the `Cause` line that we want to hide - I think we should always *at least* show this line to inform the user that the lock file was created.

The motivation behind #1347 is that the `Cause` line could often look a bit scary and lead the user to believe something went wrong, when really the line just describes *why* the lock file was created (or recreated).